superset/common/query_context.py

```diff
@@ -88,8 +88,9 @@ def __init__(
     def get_data(
         self,
         df: pd.DataFrame,
+        coltypes:[] = None
```

Comment:
nit, I think the linter would have complained, and I'd rather we be more explicit (maybe even make it required):
```suggestion
        coltypes: Optional[List[str]] = None
```

Reply:
that('s the moment you see I'm only tinkering with the code, thank's for pointing that out